Lombiq.Tests.UI/Models/SafeWaitAsync.cs

```diff
@@ -132,7 +132,7 @@ protected virtual bool DoesConditionResultSatisfy<TResult>(TResult result)
     {
         if (typeof(TResult) == typeof(bool))
         {
-            if (result is true) return true;
+            if (result.Equals(true)) return true;
         }
```

Comment:
If you use the `is` operator instead of type checking then none of this is necessary:
```csharp
        if (result is bool resultBool)
        {
            if (resultBool) return true;
        }
```